pkg/rancher-desktop/assets/translations/en-us.yaml

```diff
@@ -4742,6 +4739,43 @@ snapshots:
     icon: icon-search
     heading: No snapshots found
     body: Click on Create Snapshot to get started
+  dialog:
+    restore:
+      header: Are you sure you want to restore this Snapshot?
+      info: 'Please be aware that the current installation, including preferences, will be replaced. If the preference window is open it will be automatically closed and all unsaved changed discarded.'
+      actions:
+        ok: 'Restore'
+        cancel: 'Cancel'
+    delete:
+      header: Are you sure you want to delete this Snapshot?
+      info: 'text'
+      actions:
+        ok: 'Delete'
+        cancel: 'Cancel'
+    restoring:
+      header: 'Restoring {snapshot}'
+      message: "Please wait some minutes for the Snapshot restoration.<br/>The time depends on your machine resources.<br/>While creating the snapshot Rancher Desktop will be unavailable.<br/><br/>Once the process is finished the VM will restart, with the '<b>'{snapshot} active'</b>'."
+      actions:
+        cancel: 'Cancel'
+    creating:
+      header: 'Creating {snapshot}'
+      message: 'Please wait some minutes for the Snapshot creation.<br/>The time depends on your machine resources.<br/>While creating the snapshot Rancher Desktop will be unavailable.<br/><br/>Once the process is finished the VM will restart.'
+      actions:
+        cancel: 'Cancel'
+    buttons:
+      error: Close
+    generic:
+      header: Snapshot operation in progress
+    showLogs: Show logs
+  info:
+    restore:
+      success: "'<b>'{snapshot}'</b>' was successfully restored"
+      cancel: 'You successfully cancelled the restoration of {snapshot}'
+    create:
+      success: "'<b>'{snapshot}'</b>' was successfully created and added to the list"
+      cancel: 'You successfully cancelled the creation of {snapshot}'
+    delete:
+      success: "'<b>'{snapshot}'</b>' was successfully deleted"
```

Comment:
```suggestion
      success: "Deleted '<b>'{snapshot}'</b>'"
```

Reply:
Merged in https://github.com/rancher-sandbox/rancher-desktop/pull/5671